lib/rubocop/cop/rspec/be_nil.rb

```diff
@@ -3,34 +3,68 @@
 module RuboCop
   module Cop
     module RSpec
-      # Check that `be_nil` is used instead of `be(nil)`.
+      # Ensures a consistent style is used when matching `nil`.
       #
-      # RSpec has a built-in `be_nil` matcher specifically for expecting `nil`.
-      # For consistent specs, we recommend using that instead of `be(nil)`.
+      # You can either use the more specific `be_nil` matcher, or the more
+      # generic `be` matcher with a `nil` argument.
       #
-      # @example
+      # This cop can be configured using the `EnforcedStyle` option
       #
+      # @example `EnforcedStyle: be_nil`
```

Comment:
Adding `(default)` would be more user-friendly :-)

```suggestion
      # @example `EnforcedStyle: be_nil` (default)
```

Reply:
👍🏼 It would be nice if we had a way to check that documentation includes example for all different `EnforcedStyle`s _and_ that it marks which of them is the default.